SimG4Core/Configuration/test/dd4hep_ZMM_Run3_Step1_cfg.py

```diff
@@ -0,0 +1,203 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run3_dd4hep_cff import Run3_dd4hep
+
+process = cms.Process('SIM',Run3_dd4hep)
+
+# import of standard configurations
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('SimGeneral.HepPDTESSource.pythiapdt_cfi')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.load('Configuration.EventContent.EventContent_cff')
+process.load('SimGeneral.MixingModule.mixNoPU_cfi')
+process.load('Configuration.StandardSequences.MagneticField_cff')
+process.load('Configuration.StandardSequences.Generator_cff')
+process.load('IOMC.EventVertexGenerators.VtxSmearedRun3RoundOptics25ns13TeVLowSigmaZ_cfi')
+process.load('GeneratorInterface.Core.genFilterSummary_cff')
+process.load('Configuration.StandardSequences.SimIdeal_cff')
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+process.load('Configuration.Geometry.GeometryDD4hepExtended2021_cff') # there w
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(10),
+    output = cms.optional.untracked.allowed(cms.int32,cms.PSet)
+)
+
+# --------------------------------start of new part----------------------------
+#Other statements
+
+process.RandomNumberGeneratorService = cms.Service("RandomNumberGeneratorService",
```

Comment:
@bsunanda , our default random number generator MixMaxRng and not HepJamesRandom, also initial integer better to be different for different producers. Make all = 69 may be acceptable but better having it different.

Reply:
I am removing that part. I took the script from Sergio and made minimal change to make it work - I am removing that.